### ISSUE TITLE
Compilation issue as a dependency

### ISSUE THREAD

**Issue by lemunozm** [jcornaz/heron#124]
Hi! First of all, excellent work in this crate :)

I'm trying to compile with the provided readme version dependencies over a empty project:
```toml
bevy = "^0.5.0"
heron = { version = "0.8.0", features = ["3d"] }
```

And I get the following error coming from the `heron_debug` crate:
```
error[E0433]: failed to resolve: could not find `rapier` in `heron_rapier`
  --> /home/luis/.cargo/registry/src/github.com-1ecc6299db9ec823/heron_debug-0.8.0/src/dim2.rs:10:19
   |                                                                                                                   
10 | use heron_rapier::rapier::geometry::{ColliderHandle, ColliderSet, Shape};
   |                   ^^^^^^ could not find `rapier` in `heron_rapier`
```

I have this error only when I compile it from another project. Using `cargo build --features 2d` compiles without problems.

Thanks in advance

**Comment by jcornaz** [comment]
Hi, thanks for the report.

I think I have an idea of the root cause and how to fix it. I'll look at it asap ;-)

**Comment by jcornaz** [comment]
This should be fixed in `0.9.0`. (But that release also comes with [breaking changes](https://github.com/jcornaz/heron/releases/tag/v0.9.0)...)

Please reopen the issue if the problem persist.
